bazel/ray_deps_setup.bzl

```diff
@@ -80,18 +80,18 @@ def ray_deps_setup():
    
     http_archive(
         name = "io_opencensus_cpp",
-        strip_prefix = "opencensus-cpp-0.3.0",
-        urls = ["https://github.com/census-instrumentation/opencensus-cpp/archive/v0.3.0.zip"],
+        strip_prefix = "opencensus-cpp-3aa11f20dd610cb8d2f7c62e58d1e69196aadf11",
+        urls = ["https://github.com/census-instrumentation/opencensus-cpp/archive/3aa11f20dd610cb8d2f7c62e58d1e69196aadf11.zip"],
     )
    
     # OpenCensus depends on Abseil so we have to explicitly pull it in.
     # This is how diamond dependencies are prevented.
     git_repository(
         name = "com_google_absl",
-        commit = "88a152ae747c3c42dc9167d46c590929b048d436",
+        commit = "5b65c4af5107176555b23a638e5947686410ac1f",
```

Comment:
This commit is from March 27, right? 

https://github.com/abseil/abseil-cpp/commit/5b65c4af5107176555b23a638e5947686410ac1f

Why not a more recent one? Just curious.

Reply:
There is no special requirements to use this commit. Just it was mentioned in https://github.com/ray-project/ray/issues/4810.  The most recent one on the master branch of absl should be ok as well.